packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/converters.tests.js

```diff
@@ -162,3 +166,140 @@ testModule('Custom list', {
         });
     });
 });
+
+testModule('converter option', () => {
+    test('toHtml and fromHtml should be called once after value option changed', function(assert) {
+        const toHtmlStub = sinon.stub();
+        const fromHtmlStub = sinon.stub();
+
+        const converter = {
+            toHtml: toHtmlStub,
+            fromHtml: fromHtmlStub,
+        };
+
+        const instance = $('#htmlEditor').dxHtmlEditor({
+            converter,
+        }).dxHtmlEditor('instance');
+
+        instance.option('value', 'new value');
+
+        assert.strictEqual(toHtmlStub.callCount, 1);
+        assert.strictEqual(fromHtmlStub.callCount, 1);
+    });
+
+    test('toHtml and fromHtml must be called the correct number of times after the character has been entered', function(assert) {
+        const clock = sinon.useFakeTimers();
+        const toHtmlStub = sinon.stub();
+        const fromHtmlStub = sinon.stub();
+        const done = assert.async();
```

Comment:
assert.sync works badly together with sinon.useFakeTimers, this test can be unstable
But why in general do you need asynchrony here?